client/rpcserver/types.go

```diff
@@ -54,6 +54,25 @@ type tradeResponse struct {
 	Stamp   uint64 `json:"stamp"`
 }
 
+// myOrdersResponse is used when responding to the myorders route.
+type myOrdersResponse []*myOrder
+
+type myOrder struct {
+	Host       string `json:"host"`
+	MarketName string `json:"marketName"`
+	BaseID     uint32 `json:"baseID"`
+	QuoteID    uint32 `json:"quoteID"`
+	ID         string `json:"id"`
+	Type       string `json:"type"`
+	Sell       bool   `json:"sell"`
+	Age        uint64 `json:"age"`
+	Rate       uint64 `json:"rate"`
+	Quantity   uint64 `json:"quantity"`
+	Filled     uint64 `json:"filled"`
+	Settled    uint64 `json:"settled"`
+	Status     string `json:"status"`
```

Comment:
The following fields from `core.Order` should probably be translated (with `omitempty`).

1. `Cancelling`
2. `Canceled`
3. `TimeInForce`
4. `TargetID`

Following your convention, time-in-force can be a string, and empty for market orders. Target ID would of course be empty for all but cancel orders. 

`Rate` can maybe be `omitempty` too, since it won't apply to market orders. `Quantity`, `Filled`, and `Settled` won't apply to cancel orders.

Thoughts on having `Age` be a string? Could just use `(time.Duration).String`.

Reply:
Can I add a separate age as string? If someone were trying to use the rpcserver for automation, I just think that a number would be easier to parse. A dcrctl user would find a string more useful I believe.

About TargetID, it doesn't look like cancel orders are actually visible through the exchanges endpoint. Should I find another way to get them shown?

Also interesting, a failed cancel leaves the "cancelling" field true.

Reply:
> About TargetID, it doesn't look like cancel orders are actually visible through the exchanges endpoint

Oh, that's right. 

> Can I add a separate age as string? If someone were trying to use the rpcserver for automation, I just think that a number would be easier to parse.

Good point. You can pick how best to do this, but for computer parsing, a timestamp is conventional. That seems to be what we are using throughout.